shared/lib_battery_capacity.cpp

```diff
@@ -162,8 +162,11 @@ void capacity_t::check_SOC() {
 
             // do not switch to discharging
             state->cell_current = fmin(0, state->cell_current);
+
+            // Correct state given current change
+            state->q0 = q_upper;
         }
-        state->q0 = q_upper;
+        
```

Comment:
Was this previously incorrect or are you taking the opportunity to update formatting?

Reply:
Previously incorrect. See https://github.com/NREL/ssc/pull/1278